### ISSUE TITLE
Please stop reposting tags

### ISSUE THREAD

**Issue by alerque** [soywod/himalaya#283]
It messes everything up downstream. You can test CI / release automation in a fork if you need to. One way to do that is using a scratch org namespace that can be your own playground for this on on GitHub. You can try Actions and force push as necessary. Once a tag is posted to the canonical repository it should be considered permanent. If it was a bad release, bump the patch release number to redo it officially.

Sincerely,

Every downstream user and packager.

**Comment by law** [comment]
I just came here to complain about the release-bot spam, glad to see someone else is feeling it too.

**Comment by alerque** [comment]
I don't think the maintainer is looking at issues right now, there have been several tag reposts since I posted this. Is suspect they are buried in CI logs. Hopefully they see it when they come up for air.

**Comment by law** [comment]
I'm just going to unsubscribe for the time being.  Good luck everyone, here's hoping the maintainer sees this sometime!

**Comment by luckman212** [comment]
Came here for the CI spam like everyone else. Good luck to @soywod 
This is a great project, hope it gets sorted. Have to unsubscribe for now.

**Comment by soywod** [comment]
:fearful: I just saw your messages. I did not realize I was spamming you for all the day :cry: I'm so sorry. Himalaya is not this tiny experimental project it was back then, I should be more careful!

> You can test CI / release automation in a fork if you need to.

I will definitely do this for the next time!

Sorry again for all this mess :upside_down_face: I'm learning from my mistakes! Thank you for your advices (and your patience :pray:)
